winpr/libwinpr/sspi/Schannel/schannel.c

```diff
@@ -269,36 +247,30 @@ SECURITY_STATUS SEC_ENTRY schannel_AcceptSecurityContext(PCredHandle phCredentia
 		if (!context)
 			return SEC_E_INSUFFICIENT_MEMORY;
 
-		credentials = (SCHANNEL_CREDENTIALS*) sspi_SecureHandleGetLowerPointer(phCredential);
-
 		context->server = TRUE;
-
 		sspi_SecureHandleSetLowerPointer(phNewContext, context);
 		sspi_SecureHandleSetUpperPointer(phNewContext, (void*) SCHANNEL_PACKAGE_NAME);
-
 		schannel_openssl_server_init(context->openssl);
 	}
 
 	status = schannel_openssl_server_process_tokens(context->openssl, pInput, pOutput);
```

Comment:
If status is only used for return. Possibly we should do a:
```c
return schannel_openssl_server_process_tokens(context->openssl, pInput, pOutput);
```
And drop status completely